### ISSUE TITLE
GraphQL support broken (once again)

### ISSUE THREAD

**Issue by levrik** [hoppscotch/hoppscotch#1213]
**Describe the bug**

GraphQL requests work in general but I can't see the response from server. Neither if the request was successful or not (means contains `errors` or not).
Instead I'm just getting `SyntaxError: Unexpected token  in JSON at position 85. Check console for details.`.
With a different position depending on the actual JSON returned by server.
I think the unexpected token seems to be a space? Sadly I can't check the response since this is routed through the extension. Some debug mode for the extension would be great, that it logs request/response pairs or similar.

**To Reproduce**
1. Go to GraphQL section
2. Get the schema (works fine)
3. Enter some query or mutation
4. Send it and see this error described above.

**Expected behavior**
Returned JSON to be shown

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome
 - Version 85

**Additional context**

It's starting to get kinda "funny". Every time I'm using this application, something else is broken. Most of the time around GraphQL support, sometimes also just in general. I really like the look and feel of this application and it really makes me sad that every time I want to use some fundamental functionality is broken.

**Comment by liyasthomas** [comment]
I just tried at my end. And it's working fine. Can you produce a screenshot of a errored query/mutations?

**Comment by levrik** [comment]
![image](https://user-images.githubusercontent.com/9491603/94891637-e9d14e00-0482-11eb-8fdc-c3c7c4bfae4b.png)

The query/mutation I'm entering doesn't matter. I'm always ending up with this.

**Comment by levrik** [comment]
I can even reproduce this error with "Get schema" when using https://rickandmortyapi.com/graphql as endpoint.
For the example above, I've used our own GraphQL API. "Get schema" was working there but I'm hitting this issue on normal requests then.

**Comment by liyasthomas** [comment]
Make sure you turned off Proxy mode and Extension mode from Settings.
Make sure you're using latest GraphQL query syntax.

1. Visit https://hoppscotch.io/graphql
2. Endpoint: https://rickandmortyapi.com/graphql
3. Query:

```
query GetCharacter($id: ID!) {
  character(id: $id) {
    id
    name
  }
}
```
4. Send

If this doesn't work, try cleaning site's cache. And try again.

**Comment by levrik** [comment]
@liyasthomas Yep. This is working fine so the problem is probably the extension.
